tidy3d/plugins/microwave/models/coupled_microstrip.py

```diff
@@ -51,7 +50,7 @@ def _z0_even_odd(
     e_eff_odd: float,
     z0: float,
     e_eff: float,
-) -> Tuple[float, float]:
+) -> tuple[float, float]:
```

Comment:
Use `Tuple` to be consistent with the rest of code?

Reply:
I think we are expecting to deprecate using those aliases. [see here](https://docs.python.org/3/library/typing.html#deprecated-aliases)

Is that right @tylerflex ?

Reply:
yea we should start using the builtins like `tuple`